tests/topotests/lib/bgp.py

```diff
@@ -716,7 +716,7 @@ def __create_bgp_neighbor(topo, input_dict, router, addr_type, add_neigh=True):
     """
     config_data = []
     logger.debug("Entering lib API: {}".format(sys._getframe().f_code.co_name))
-
+    tgen = get_topogen()
```

Comment:
These changes need to be in the previous commit.

Reply:
Cleaned up the commit message and moved the changes to proper commits.